rfcs/remote_channel.md

```diff
@@ -0,0 +1,616 @@
+# RFC 98: Cross-Window Channels
+
+## Summary
+
+Provide a channel API which provides support for server-mediated
+messaging between browsing contexts, including those which are in
+different browsing context groups, and so cannot communicate purely
+from the client.
+
+## Details
+
+### Use Case
+
+The modern web platform has features which allow isolating different
+browsing contexts and scripting environments from each other such that
+they are unable to communicate. For example windows opened with the
+`noopener` attribute will not have a handle to their parent, nor will
+the parent have a handle to the child. Similarly, cross-origin
+navigations with the `cross-origin-opener-policy` header appropriately
```

Comment:
Is this RFC going to replace the framework in the COOP/COEP tests?

Reply:
I don't expect the RFC to require moving the COOP/COEP tests. However I expect the RFC to provide all the features needed for those tests, and would anticipate the actual migration happening as a followup once the implementation has landed.